cmd/endpoint.go

```diff
@@ -18,16 +18,15 @@ package cmd
 
 import (
 	"fmt"
-	"net"
-	"net/url"
+	gonet "net"
```

Comment:
better not change the names of stdlib and instead use minionet:
`minionet "github.com/minio/minio/pkg/net"`

this way we need not worry about which package "net" refers to in our codebase when reading source

Reply:
better, we can rename github.com/minio/minio/pkg/net to github.com/minio/minio/pkg/minionet